ionoscloud/data_source_dbaas_mongo_template.go

```diff
@@ -24,11 +25,13 @@ func dataSourceDbassMongoTemplate() *schema.Resource {
 				Description:      "The unique ID of the template.",
 				ValidateDiagFunc: validation.ToDiagFunc(validation.IsUUID),
 				Optional:         true,
+				Computed:         true,
```

Comment:
@cristiGuranIonos for many data sources that have the feature to retrieve using both `ID` and the `name` we have `ID` and `name` marked as `optional` but not as `computed`. Do we need to make this change for all the data sources?

Reply:
might be, but we can wait until someone raises an issue